Refactor views by using generic view

harinath01 opened the pull request:
> Used generic views to abstract:
      - getting data from the database according to a parameter passed in the URL
      - loading a template
      - returning the rendered template

harinath01 committed:
Refactor views'

waranlogesh commented on mysite/polls/urls.py:
```diff
@@ -1,10 +1,10 @@
 from django.urls import path
-from .views import index, detail, results, vote
+from .views import *
```
Import * shouldn't be used

waranlogesh commented on mysite/polls/urls.py:
```diff
@@ -1,10 +1,10 @@
-    path("<int:question_id>/", detail, name="detail"),
-    path("<int:question_id>/results", results, name="results"),
-    path("<int:question_id>/vote", vote, name="vote"),
+    path('', IndexView.as_view(), name='index'),
```
Undo the quotes change

harinath01 committed:
Update urls.py

- Used double quotes instead of single quotes
- And imported the views explicitly.
